apps/server/lib/google-drive/src/resources/files/permissions.ts

```
@@ -0,0 +1,1066 @@
+export namespace PermissionDeleteParams {
+	export interface _ {
+		/**
+		 * V1 error format.
+		 */
+		xgafv?: "1" | "2";
+	}
+}
+
+export declare namespace Permissions {
+	export {
+		type Permission as Permission,
```

Biome (1.9.4)

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)